tests/uvdata/test_mwa_corr_fits.py

```diff
@@ -1074,7 +1081,7 @@ def test_van_vleck_cheby():
     good_ants = np.delete(np.unique(uv2.ant_1_array), 76)
     uv2.select(antenna_nums=good_ants)
 
-    assert np.allclose(uv1.data_array, uv2.data_array)
+    assert uv1._data_array == uv2._data_array
```

Comment:
Sorry I didn't specifically ping this one but I think it falls under the same comment about using the numpy utility.